CHANGELOG.md

```diff
@@ -0,0 +1,886 @@
+  - Changes to logging within sync framework to ensure consistency of logging 
+  - Marks pending updates as failed where a datasetclient has been removed from the db (via a cleanup task for inactive clients) yet there are still pending items in the queue for that datasetclient, as we don't have enough info to process it correctly.
+  - Formats comments as proper jsdoc
+  - Change to not lock the datasetclient collection on sync api call
+  - Adds new config option (sync)
+  - Change to use debug module for logging (sync)
+  - Changed to use updateOne instead of deprecated update
+ 
+
+### Removed
+- Removes the start API from the sync server
+
+
+## [6.1.8] - Wed Mar 15 11:49:24 2017 +0000
+### Changed
+- updates dependencies to mitigate qs and moment vulnerabilities
+```git
+-    "fh-mbaas-client": "0.15.0",
+-    "fh-mbaas-express": "5.6.9",
++    "fh-mbaas-client": "0.16.5",
++    "fh-mbaas-express": "5.6.10",
+-    "moment": "2.14.1",
++    "moment": "2.15.2",
+-    "request": "2.74.0",
++    "request": "2.81.0",
+-    "unifiedpush-node-sender": "0.12.0",
++    "unifiedpush-node-sender": "0.12.1",
+```
+
+
+## [6.1.7] - Wed Mar 15 14:31:38 2017 +0000
+### Changed
+- Fixes travis CI build dependency on docker by adding to 'services'
+- updated fh-db as couldn't import a previously exported empty json collection within the Studio's Data Browser:
+```git
+-    "fh-db": "1.2.5",
++    "fh-db": "1.4.4", 
+```
+
+
+## [6.1.6] - Mon Mar 6 17:24:38 2017 +0000
+### Added
+- Added support for custom hashing rules for sync records
+
+### Changed
+- Integrated sync-acceptance-testing into Travis build
+- Fixed bug where databrowser export doesn't work if database not upgraded
+
+
+## [6.1.5] - Wed Feb 1 13:41:50 2017 +0100
+### Added
+- Added support for response handler (sync)
+- Added test for responseInterceptor
+
+### Changed
+- updated dependency:
+```git
+-    "fh-mbaas-express": "5.6.8",
++    "fh-mbaas-express": "5.6.9",
+```
+
+
+## [6.1.4] - Thu Jan 19 10:52:27 2017 +0000
+### Added
+- Adds extra sync tests to 'unit' test run
+- Adds note to readme re multiple sync loop caveat
+
+### Changed
+- Change in logging level (from `error` to `info`)
+
+
+## [6.1.3] - Wed Jan 4 18:01:56 2017 +0000
+### Changed
+- version number of fhm-mbaas-api from 6.1.2 to 6.1.3 to republish to NPM
+
+
+## [6.1.2] - Wed Jan 4 09:50:40 2017 +0000
+### Changed
+- update of fh-mbaas-express dependency to include Insecure Entropy Source fix
+```git
+-    "fh-mbaas-express": "5.6.4",
++    "fh-mbaas-express": "5.6.6",
+```  
+
+
+## [6.1.1] - Fri Dec 9 16:15:37 2016 +0000
+### Added
+- Adds new auth endpoint
+
+
+## [6.1.0] - Sun Nov 27 20:56:21 2016 +0000
+### Changed
+- updated dependency:
+```git
+-    "fh-db": "1.2.3",
++    "fh-db": "1.2.5",
+```
+
+
+## [6.0.1] - Tue Nov 15 16:05:20 2016 +0000
+### Changed
+- version number of fhm-mbaas-api from 6.0.0 to 6.0.1 to republish to NPM
+
+
+## [6.0.0] - Mon Nov 14 14:53:36 2016 +0000
```

Comment:
@damienomurchu Might be good just to point out explicitly in a note here like you did for v7.0.0 that the major version bump was due to the removal of the .feed api.

Reply:
Good points - both included now

Reply:
thannks @damienomurchu